[2022-08-06 19:37:22,217] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-12-08T00:00:00+00:00 [queued]>
[2022-08-06 19:37:22,234] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-12-08T00:00:00+00:00 [queued]>
[2022-08-06 19:37:22,234] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 19:37:22,234] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 19:37:22,234] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 19:37:22,255] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2021-12-08 00:00:00+00:00
[2022-08-06 19:37:22,262] {standard_task_runner.py:52} INFO - Started process 1344 to run task
[2022-08-06 19:37:22,265] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-12-08T00:00:00+00:00', '--job-id', '399', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp3aheqx7m', '--error-file', '/tmp/tmpoybkeogu']
[2022-08-06 19:37:22,265] {standard_task_runner.py:80} INFO - Job 399: Subtask second_task
[2022-08-06 19:37:22,350] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-12-08T00:00:00+00:00 [running]> on host b29c168c1666
[2022-08-06 19:37:22,447] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-12-08T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-08T00:00:00+00:00
[2022-08-06 19:37:22,448] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 19:37:22,448] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 19:37:22,460] {subprocess.py:85} INFO - Output:
[2022-08-06 19:37:22,461] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 19:37:22,461] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 19:37:22,495] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20211208T000000, start_date=20220806T193722, end_date=20220806T193722
[2022-08-06 19:37:22,556] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 19:37:22,595] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 21:51:14,824] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-12-08T00:00:00+00:00 [queued]>
[2022-08-06 21:51:14,836] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-12-08T00:00:00+00:00 [queued]>
[2022-08-06 21:51:14,837] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:51:14,837] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 21:51:14,837] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:51:14,852] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2021-12-08 00:00:00+00:00
[2022-08-06 21:51:14,858] {standard_task_runner.py:52} INFO - Started process 1525 to run task
[2022-08-06 21:51:14,861] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-12-08T00:00:00+00:00', '--job-id', '398', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpmabbn9_5', '--error-file', '/tmp/tmpw8yg430b']
[2022-08-06 21:51:14,862] {standard_task_runner.py:80} INFO - Job 398: Subtask second_task
[2022-08-06 21:51:14,926] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-12-08T00:00:00+00:00 [running]> on host c3f731879890
[2022-08-06 21:51:15,011] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-12-08T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-08T00:00:00+00:00
[2022-08-06 21:51:15,011] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 21:51:15,012] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 21:51:15,024] {subprocess.py:85} INFO - Output:
[2022-08-06 21:51:15,025] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 21:51:15,025] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 21:51:15,054] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20211208T000000, start_date=20220806T215114, end_date=20220806T215115
[2022-08-06 21:51:15,072] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 21:51:15,102] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 21:56:41,589] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-12-08T00:00:00+00:00 [queued]>
[2022-08-06 21:56:41,602] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-12-08T00:00:00+00:00 [queued]>
[2022-08-06 21:56:41,602] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:56:41,602] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 21:56:41,602] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:56:41,619] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2021-12-08 00:00:00+00:00
[2022-08-06 21:56:41,632] {standard_task_runner.py:52} INFO - Started process 1324 to run task
[2022-08-06 21:56:41,635] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-12-08T00:00:00+00:00', '--job-id', '399', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp2pwoj0mk', '--error-file', '/tmp/tmpneq92m_q']
[2022-08-06 21:56:41,635] {standard_task_runner.py:80} INFO - Job 399: Subtask second_task
[2022-08-06 21:56:41,706] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-12-08T00:00:00+00:00 [running]> on host bc7e154a4fdd
[2022-08-06 21:56:41,800] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-12-08T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-08T00:00:00+00:00
[2022-08-06 21:56:41,801] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 21:56:41,801] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 21:56:41,814] {subprocess.py:85} INFO - Output:
[2022-08-06 21:56:41,816] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 21:56:41,816] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 21:56:41,858] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20211208T000000, start_date=20220806T215641, end_date=20220806T215641
[2022-08-06 21:56:41,926] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 21:56:41,994] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 22:11:43,691] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-12-08T00:00:00+00:00 [queued]>
[2022-08-06 22:11:43,706] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-12-08T00:00:00+00:00 [queued]>
[2022-08-06 22:11:43,706] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:11:43,706] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 22:11:43,706] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:11:43,720] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2021-12-08 00:00:00+00:00
[2022-08-06 22:11:43,726] {standard_task_runner.py:52} INFO - Started process 1335 to run task
[2022-08-06 22:11:43,728] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-12-08T00:00:00+00:00', '--job-id', '399', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmprk0_6igx', '--error-file', '/tmp/tmp66tecn3l']
[2022-08-06 22:11:43,728] {standard_task_runner.py:80} INFO - Job 399: Subtask second_task
[2022-08-06 22:11:43,796] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-12-08T00:00:00+00:00 [running]> on host 1250c3b659cd
[2022-08-06 22:11:43,875] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-12-08T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-08T00:00:00+00:00
[2022-08-06 22:11:43,875] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 22:11:43,876] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 22:11:43,888] {subprocess.py:85} INFO - Output:
[2022-08-06 22:11:43,889] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 22:11:43,889] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 22:11:43,920] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20211208T000000, start_date=20220806T221143, end_date=20220806T221143
[2022-08-06 22:11:43,940] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 22:11:43,966] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 22:23:03,834] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-12-08T00:00:00+00:00 [queued]>
[2022-08-06 22:23:03,851] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-12-08T00:00:00+00:00 [queued]>
[2022-08-06 22:23:03,851] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:23:03,851] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 22:23:03,851] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:23:03,870] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2021-12-08 00:00:00+00:00
[2022-08-06 22:23:03,879] {standard_task_runner.py:52} INFO - Started process 1386 to run task
[2022-08-06 22:23:03,881] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-12-08T00:00:00+00:00', '--job-id', '401', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp3zeniu5o', '--error-file', '/tmp/tmp_z_5e_01']
[2022-08-06 22:23:03,882] {standard_task_runner.py:80} INFO - Job 401: Subtask second_task
[2022-08-06 22:23:03,985] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-12-08T00:00:00+00:00 [running]> on host b215076695c3
[2022-08-06 22:23:04,092] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-12-08T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-08T00:00:00+00:00
[2022-08-06 22:23:04,093] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 22:23:04,094] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 22:23:04,113] {subprocess.py:85} INFO - Output:
[2022-08-06 22:23:04,116] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 22:23:04,117] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 22:23:04,202] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20211208T000000, start_date=20220806T222303, end_date=20220806T222304
[2022-08-06 22:23:04,254] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 22:23:04,335] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
